crates/gitbutler-core/src/git/diff.rs

```diff
@@ -124,7 +123,11 @@ impl GitHunk {
 #[serde(rename_all = "camelCase")]
 pub struct HunkLock {
     pub branch_id: Id<Branch>,
-    pub commit_id: git::Oid,
+    #[serde(
+        serialize_with = "crate::serde::oid::serialize",
+        deserialize_with = "crate::serde::oid::deserialize"
```

Comment:
```suggestion
with = "crate::serde::oid"
```
https://serde.rs/field-attrs.html#with

Reply:
thanks!